src/index.ts

```diff
@@ -177,11 +179,13 @@ export class UnleashClient extends TinyEmitter {
         this.refreshInterval = disableRefresh ? 0 : refreshInterval * 1000;
         this.context = { appName, environment, ...context };
         this.usePOSTrequests = usePOSTrequests;
+        this.sdkError = null;
```

Comment:
what is we had a union with a current SDK state: type SdkState = 'healthy' | 'error' or similar?

Reply:
I like it and for the event name, may I suggest `RECOVERED` or `RECOVERED_FROM_ERROR`?